### ISSUE TITLE
Hyperparameter optimization ?

### ISSUE THREAD

**Issue by debasmitdas** [RockySJ/WDGRL#5]
In the experiments, do you report the best results obtained over the hyper-parameter range or do you do some kind of validation. Validation is generally not possible for unsupervised domain adaptation.

**Comment by RockySJ** [comment]
We follow Long et al. (Transfer Feature Learning with Joint Distribution Adaptation) and evaluate all compared approaches through grid search on the hyperparameter space, and report the best results of each approach. 
BTW, Bousmalis et al. (Domain Separation Network) discuss the hyperparameter tuning issue in the unsupervised domain adaptation setting and use a small set of labeled target domain data as a validation set.
